addons/bus/static/tests/mock_server/mock_models/bus_bus.js

```diff
@@ -89,7 +89,10 @@ export class BusBus extends models.Model {
         }
         for (const notification of notifications) {
             const [type, payload] = notification.slice(1, notification.length);
-            values.push({ id: ++this.lastBusNotificationId, message: { payload, type } });
+            values.push({
+                id: ++this.lastBusNotificationId,
+                message: { payload: JSON.parse(JSON.stringify(payload)), type },
```

Comment:
Note for reviewer: not a necessary change, but it allows detecting issues (passing incorrect values) and gets closer to python processing (also json).